.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 dist: trusty
 language: php
 php:
+    - 5.6
```

Comment:
This isn't needed if our nextcloud version doesn't support it.

Reply:
I know. Just added for testing. Will be removed with next push.